release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+- [browser#838](https://github.com/grafana/xk6-browser/pull/838) Removed the first meaningful paint metric. This metric is being deprecated across all the browsers, because the metric's definition relies on browser-specific implementation details, and we've now introduced web vitals in the browser module which is a reliable industry standard way to measure frontend performance. You can find more details [here](https://developer.chrome.com/en/docs/lighthouse/performance/first-meaningful-paint/).
+- [browser#843](https://github.com/grafana/xk6-browser/pull/843) Removed the build step from Github Actions. From this release onwards, no new standalone browser binaries will be built and available from the [releases](https://github.com/grafana/xk6-browser/releases) section. The latest version of the browser module will be available in the k6 binary which can be found in the [k6 releases](https://github.com/grafana/k6/releases) page.
+
+## New features
+
+### A new experimental module implementing the Web Crypto API specification [#3007](https://github.com/grafana/k6/pull/3007)
+
+This release includes a new experimental `webcrypto` module partially implementing the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification in k6.
+
+<details>
+<summary> Expand to see an example of the new functionality.</summary>
+
+This example shows encrypting and decrypting of a "Hello, World!" string using AES-CBC algorithm.
+
+```javascript
+import { crypto } from 'k6/experimental/webcrypto';
+
+export default async function () {
+  const key = await crypto.subtle.generateKey(
+    {
+      name: 'AES-CBC',
+      length: 256,
+    },
+    true,
+    ['encrypt', 'decrypt']
+  );
+
+  const encoded = stringToArrayBuffer('Hello, World!');
+  const iv = crypto.getRandomValues(new Uint8Array(16));
+
+  const ciphertext = await crypto.subtle.encrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    encoded
+  );
+
+  const plaintext = await crypto.subtle.decrypt(
+    {
+      name: 'AES-CBC',
+      iv: iv,
+    },
+    key,
+    ciphertext
+  );
+
+  console.log(
+    'deciphered text == original text: ',
+    arrayBufferToHex(plaintext) === arrayBufferToHex(encoded)
+  );
+}
+
+function arrayBufferToHex(buffer) {
+  return [...new Uint8Array(buffer)].map((x) => x.toString(16).padStart(2, '0')).join('');
+}
+
+function stringToArrayBuffer(str) {
+  var buf = new ArrayBuffer(str.length * 2); // 2 bytes for each char
+  var bufView = new Uint16Array(buf);
+  for (var i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+}
+```
+</details>
+
+You can see the list of currently supported APIs and algorithms in the project's [README](https://github.com/grafana/xk6-webcrypto/blob/v0.1.0/README.md#current-state).
+
```

Comment:
```suggestion

Docs: k6-docs link here
```
We should add the k6-docs link

Reply:
:+1: Added a sentence in the latest commit